config/checkstyle/checkstyle.xml

```diff
@@ -30,7 +30,7 @@
             <property name="allowNonPrintableEscapes" value="true"/>
         </module>
         <module name="LineLength">
-            <property name="max" value="120"/>
+            <property name="max" value="160"/>
             <property name="ignorePattern" value="^package.*|^import.*|a href|href|http://|https://|ftp://"/>
```

Comment:
Code is difficult to review here once line length exceed github code window size

Reply:
Hmm you can insert this in console so you have wider diffs in github:
`
javascript: document.body.classList.toggle('full-width');
`

Reply:
You can actually make a bookmark in your browser and click it to make the diffs the width of the browser window.